index.d.ts

```diff
@@ -1446,22 +1446,28 @@ declare namespace Eris {
 
   export class Invite implements SimpleJSON {
     code: string;
-    channel: { id: string; name: string };
+    channel: {
```

Comment:
Similar thoughts on this as well since it is a partial channel are we certain that all these props will exist?

Reply:
Channels will always have a type and id, if it's an invite to a group DM name will be null, and icon will exist, so I'll update that

Reply:
name will be null if the group dm doesn't have a name*